applications/ChatGPT/chatgpt/dataset/sft_dataset.py

```diff
@@ -44,6 +44,8 @@ def __getitem__(self, idx):
 
 
 class DistributedSampler:
+    # This code is borrowd from:
```

Comment:
https://pytorch.org/docs/stable/_modules/torch/utils/data/distributed.html#DistributedSampler
Can we use torch distributed sampler here?
Or if you want to use Wenet's code, we have to add license into our license and declare the copyright.

Reply:
I have rewritten the sampler with torch distributed sampler